exe/dirtree

```diff
@@ -43,13 +46,19 @@ OptionParser.new do |opts|
     options[:local_dependencies] = true
   end
 
+  opts.on(
+    '-s',
+    '--screenshot',
+    'Get an image screenshot of the directory tree'
+  )
+
   opts.on(
     '-oFile.html',
     '--output=File.html',
     'Specify a path to write output, if
        not specified output will be printed to STDOUT'
   ) do |value|
-    options[:output] = value
+    options[:output] = value.empty? ? default_file_name : value
```

Comment:
What should this line do? 
my understanding to the block is that it's invoked when `-o` or `--output` flag is provided.

so here we're checking is `-o` is provided without a value after it?

Reply:
I think there is entanglement here between the `output` feature, that will output HTML file for you, and the `screenshot` feature that output an image for you.
I think the `options[:output]` value should be as is without any change.

Reply:
This change wasn't made for the screenshot functionality.
The program had an issue that if someone used the output flag `-o` without specifying a name name the program will just stuck there without doing any further action so I think filling this variable with a default value will solve this issue.

Reply:
In that case the fix is just exiting with a message if the flag was provided without a value.